catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -1130,7 +1130,9 @@ public void setHistorianConfigurations(List<HistorianConfiguration> historians)
   }
 
   public void bind(HistorianConfiguration historianConfiguration) {
-    this.historianConfiguration = historianConfigurations.get(0);
+    if (this.historianConfigurations != null && !this.historianConfigurations.isEmpty()) {
+      this.historianConfiguration = historianConfigurations.get(0);
+    }
```

Comment:
❓ @rzwiefel or @glenhein I noticed rearranging the blueprints did expose an NPE here when watching the bundle and rebuilding. I added protections but wanted to make sure there's nothing else we wanted to do. We seem to be managing multiple historian configurations in an awkward way.

Reply:
Yeah this probably shouldn't be a reference list at all, just a single service.

Reply:
Sounds good 👍 I can update.

Reply:
❓ Is there a reason we aren't using the `historianConfiguration` parameter here?

Reply:
I'm wondering that too. Probably worth retesting the historian. Updating test instructions.